### DISCUSSION TITLE
Groups Made Easy (IdAM)

### DISCUSSION THREAD

**Discussion by lxs** [tripupapp/tripup-server#6] [Ideas]
**Gist:** Ability to assign group access to photo albums. 

**Background:** a 'joy' of Google Picassa was the integration with groups defined in user's Google Contacts. This took away excessive hassle of managing access rights. (e.g. family were  members of "Family Group" in Google Contacts, and could be granted access to specific albums using the group, rather than defining individual access / revoking individual access).
-- Yet, Google. Preference for robust privacy. 
-- Also, the service and feature are no longer available.

**Furthermore:** this would also allow group notifications to be sent when new photos are added, or comments are made on photos within the album (assuming such a feature were created)

**Comment by vin047** [comment]
I like this idea but I will have to table this for the moment, until TripUp has amassed more users and people start experiencing large group lists as a pain point.
